phpBB/styles/prosilver/theme/common.css

```diff
@@ -3,8 +3,6 @@
 /* -------------------------------------------------------------- */
 /* stylelint-disable selector-max-compound-selectors */
 /* stylelint-disable selector-no-qualifying-type */
-/* stylelint-disable declaration-property-unit-whitelist */
-/* stylelint-disable declaration-property-unit-blacklist */
```

Comment:
not just the css but this was just an example as posting_editor.html is old 3.2 for sure